transaction/std/src/memo/authenticated_common.rs

```diff
@@ -64,7 +31,7 @@ pub fn validate_authenticated_sender(
         shared_secret.as_ref(),
         tx_out_public_key,
         memo_type_bytes,
-        memo_data,
+        memo_data[..48].try_into().unwrap(),
     );
     let found_hmac: [u8; 16] = memo_data[(64 - 16)..].try_into().unwrap();
```

Comment:
:wrench: mixing `48` and `64-16` 
```suggestion
        memo_data[..48].try_into().unwrap(),
    );
    let found_hmac: [u8; 16] = memo_data[48..].try_into().unwrap();
```